packages/ffe-form/less/radio-block.less

```diff
@@ -8,21 +8,21 @@
 //             aria-invalid="false">
 //         <div class="ffe-radio-block__content">
 //             <label class="ffe-radio-block__header" for="radio-block-1">Du</label>
-//             <div class="ffe-radio-block__wrapper ffe-radio-block__wrapper--empty"></div>
+//             <div class="ffe-radio-block__wrapper ffe-radio-block__content__wrapper--empty"></div>
 //         </div>
 //     </div>
 //     <div class="ffe-radio-block">
 //         <input type="radio" class="ffe-radio-input" id="radio-block-2" name="owner" value="partner"
 //             aria-invalid="false">
 //         <div class="ffe-radio-block__content">
-//             <label class="ffe-radio-block__header" for="radio-block-2">Ektefelle, samboer eller registrert partner</label>
+//             <label class="ffe-radio-block__content__header" for="radio-block-2">Ektefelle, samboer eller registrert partner</label>
 //         </div>
 //     </div>
 //     <div class="ffe-radio-block">
 //         <input type="radio" class="ffe-radio-input" id="radio-block-3" name="owner" value="leasing-company"
 //             aria-invalid="false">
 //         <div class="ffe-radio-block__content">
-//             <label class="ffe-radio-block__header" for="radio-block-3">Leasingselskap</label>
+//             <label class="ffe-radio-block__content__header" for="radio-block-3">Leasingselskap</label>
```

Comment:
I BEM bør hvert element være uavhengig av andre elementer, altså bør dobbel underscore som tommelfingerregel ikke gjentas i klasser. `__header` og `__wrapper` trenger med andre ord ikke å være avhengige av `__content` i tillegg til selve blocken. Denne nøstingen legger føringer for markupen som kan gjøre det vanskeligere å senere endre på strukturen i komponenten. Og `__header` kan fortsatt brukes inni `__content`. :)

Se [Why not create elements of elements (block__elem1__elem2)?](https://en.bem.info/methodology/faq/#why-not-create-elements-of-elements-block__elem1__elem2)

Reply:
Ahaa! Da går jeg bare tilbake til sånn classNamesene var satt opp. Takk :)